core/web/loader/loader.go

```diff
@@ -45,6 +46,7 @@ func New(app chainlink.Application) *Dataloader {
 		app: app,
 
 		ChainsByIDLoader:                          dataloader.NewBatchedLoader(chains.loadByIDs),
```

Comment:
Is there a ticket to clean out the `ChainsByIDLoader` usage?

Reply:
Not yet, there is still code using it for backwards compatibility as it is exposed via API, and to be honest i dont know enough about how this repo works and removing this is a breaking change for the API, like we would have to communicate to consumers to not use this or monitor the usage of it.